Library/Homebrew/cask/artifact/installer.rb

```diff
@@ -15,7 +15,7 @@ module ManualInstaller
         def install_phase(**)
           puts <<~EOS
             To complete the installation of Cask #{cask}, you must also
-            run the installer at
+            run the installer at:
 
```

Comment:
Thoughts on trimming the extra empty line here?

Reply:
Solid idea; seems wise to save line breaks for trailing important messages.